setup.cfg

```diff
@@ -56,3 +56,6 @@ max-line-length = 100
 
 [pycodestyle]
 max-line-length = 100
+
+[bdist_wheel]
+py_limited_api = cp37
```

Comment:
I bumped Python minversion to 3.9 in #107 , so this needs updating. FYI

Reply:
Done